### PR TITLE
[ci] Update to Common CI Script

### PR DESCRIPTION
**Pull request by talha-p**
Updates openwisp-utils to latest version
Migrates to the common CI script provided in openwisp-utils (https://github.com/openwisp/openwisp-utils/pull/45)

### CODE REVIEW

**Comment by talha-p** [comment]
I'm a bit unsure on why Travis is failing?
Its a segmentation fault caused when using coverage on the SAMPLE_APP build

File: requirements.txt [hunk 1]
```diff
@@ -1,5 +1,5 @@
 django>=1.11,<2.2
 swapper>=1.1.0
-openwisp-utils>=0.2,<0.3
+openwisp-utils>=0.3.0
```
Comment (atb00ker) [inline requirements.txt]: Can you please do it like before?
`openwisp-utils>=0.3,<0.4` so that we can have a lower and upper limit not just lower limit.

**Review by atb00ker — COMMENTED** [review]
Hi,

Travis error seems not be not related to your work, you can ignore it. :smile:
Please read the comment below.

File: requirements.txt (hunk 1 above)
Comment (talha-p) [reply requirements.txt]: @atb00ker Changed

**Review by atb00ker — COMMENTED** [review]
Tested it on my machine, travis failing should be a travis related issue not due to this PR.

**Review by nemesifier — APPROVED** [review]

**Pull request merged**